packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -125,8 +124,13 @@ exports.onPostBuild = (args, pluginOptions) => {
       const swAppend = fs
         .readFileSync(`${__dirname}/sw-append.js`, `utf8`)
         .replace(/%pathPrefix%/g, pathPrefix)
-
       fs.appendFileSync(`public/sw.js`, `\n` + swAppend)
+
+      if (pluginOptions.injectScript) {
+        const userAppend = fs.readFileSync(pluginOptions.injectScript, `utf8`)
+        fs.appendFileSync(`public/sw.js`, `\n` + userAppend)
```

Comment:
What do you think of wrapping this in a try catch here and printing a friendly error if the file doesn't exist? Definitely not a blocker for this PR though.

Reply:
Agreed, I'll implement this change

Reply:
ok, just done this!

Reply:
Hi there, sorry for intruding, but I'm wondering about the status of this (not super familiar with the code review interface on GH, but it appears to have passed review?). 

Anybody know if this will be implemented soon, or if it's still waiting on something? Or is there anything I can help with to move it along? 

I'm currently working on a project that would benefit greatly from being able to extend functionality of the plugin's service worker!

Thanks in advance 🙂